source/WanManager/wanmgr_interface_sm.c

```diff
@@ -372,7 +372,6 @@ static BOOL WanMgr_RestartFindExistingLink (WanMgr_IfaceSM_Controller_t* pWanIfa
             //TODO : NEW_DESIGN check VLAN status for Cellular
             p_VirtIf->VLAN.Status = WAN_IFACE_LINKSTATUS_UP;
 
-            strncpy(p_VirtIf->Name, CELLULARMGR_WAN_NAME, sizeof(p_VirtIf->Name));
```

Comment:
Before this API call, p_VirtIf->Name has correct interface from PSM DB. So, no need to overwrite here.